superset/utils/network.py

```diff
@@ -38,8 +38,6 @@ def is_port_open(host: str, port: int) -> bool:
     finally:
         s.close()
 
-    return False
```

Comment:
bycatch: I notice that `port` is an `int ` in the argument list, but we're recasting it to `int` on line 33, which is unnecessary. While we're at it, I'd almost recommend removing that cast, and make sure code ever calls this function with anything except an `int` value.

Reply:
Good spot; I have taken out the `int` cast - by the looks of it, `is_port_open()` is only used [here](https://github.com/apache/superset/blob/master/superset/db_engine_specs/base.py#L1544), where `port` is indeed an integer.